client/src/ui/diagram.html

```diff
@@ -1,61 +1,58 @@
 <html lang="en">
 
 <head>
+    <meta charset="UTF-8">
+
     <link href="{pathCss}/bootstrap.min.css" rel="stylesheet"
           integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link href="{pathCss}/fontawesome.css" rel="stylesheet" crossorigin="anonymous">
     <link href="{pathCss}/simpleXML.css" rel="stylesheet" crossorigin="anonymous">
     <link href="{pathCss}/custom.css" rel="stylesheet" crossorigin="anonymous">
+    <link href="{pathCss}/jsonview.css" rel="stylesheet" crossorigin="anonymous">
+
     <script src="{pathJS}/jquery-3.4.1.js"></script>
+    <script src="{pathJS}/popper.js"></script>
+    <script src="{pathJS}/fontawesome.js"></script>
     <script src="{pathJS}/jsplumb.min.js"></script>
     <script src="{pathJS}/simpleXML.js"></script>
-
```

Comment:
Without having two different path variables such as `{pathCss}` and `{pathJS}` use single path variable to a location such as `assets` or similar folder name and then deviate from that path. 

Example : `<link href="{resourcePath}/style/fontawesome.css" rel="stylesheet" crossorigin="anonymous">`

Reply:
Fixed